responsibleai/text/components/rai_text_insights/spec.yaml

```diff
@@ -1,59 +1,75 @@
 $schema: http://azureml/sdk-2-0/CommandComponent.json
 name: rai_text_insights
 display_name: RAI Text Insights
-version: 0.0.7
+version: 0.0.8
 tags:
   Preview: ""
 type: command
 inputs:
   task_type:
     type: string # [text_classification]
     enum: ['text_classification', 'multilabel_text_classification']
+    description: The type of task to perform
   model_input: # mlflow model name:version
     type: mlflow_model
     optional: false
+    description: The model to be used for computing insights
   model_info:
     type: string # model name:version
     optional: false
+    description: The model to be used for computing insights
```

Comment:
Should this be model name in the description?

Reply:
done!